src/dsp_tools/excel2xml.py

```diff
@@ -444,7 +445,7 @@ def make_bitstream_prop(
                       stacklevel=2)
     prop_ = etree.Element("{%s}bitstream" % (xml_namespace_map[None]), permissions=permissions,
```

Comment:
It's a detail, but I'm curious: why do you append a `_` to `prop_`? Is prop a reserved name in Python?